workshop/content/20-typescript/20-create-project/500-deploy.md

```diff
@@ -8,8 +8,8 @@ Okay, we've got a CloudFormation template. What's next? __Let's deploy it into o
 ## Bootstrapping an environment
 
 The first time you deploy an AWS CDK app into an environment (account/region),
-you'll need to install a "bootstrap stack". This stack includes resources that
-are needed for the toolkit's operation. For example, the stack includes an S3
+you can install a "bootstrap stack". This stack includes resources that
```

Comment:
This is good wording, so I think its worth merging, but just to be clear, bootstrapping is required to deploy any stack that uses Assets, Pipelines, or is above a certain size.